### ISSUE TITLE
User count feature broken?

### ISSUE THREAD

**Issue by amitschandillia** [strapi/strapi#4232]
**Describe the bug**
I used to be able to perform a `count` query on the `user` document in **alpha**. Now that I've updated to **beta**, that feature seems to have disappeared.

**Steps to reproduce the behavior**
1. Go to 'https://dev.schandillia.com/graphql'
2. Run the following query:
```
{
 usersConnection{
    aggregate{
      count
    }
  }
}
```
3. See error: `The model user can't be found`.

**Expected behavior**
I expected it to return the count of `users`, say, 3 (in my case). This was the behavior in `alpha`.

**Screenshots**
![strapi-users](https://user-images.githubusercontent.com/42805381/66574367-b884b180-eb91-11e9-9a30-f705ac8f10eb.jpg)

**Code snippets**
_Not applicable_, but repo up at https://github.com/amitschandillia/proost/tree/master/dev for reference.

**System**
- Node.js version: **10.16.0**
- NPM version: **6.9.0**
- Strapi version: **3.0.0-beta.16.8**
- Database: **MongoDB**
- Operating system: **Ubuntu**

**Additional context**
When I checked **Roles and Permissions** -> **USER PERMISSIONS** -> **Users**, I found that the `count` route is missing (see screenshot above).

**Comment by derrickmehaffy** [comment]
There is no user count route: https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/config/routes.json

**Comment by amitschandillia** [comment]
Exactly. But there used to be one in the alpha version. Is there any workaround possible?

**Comment by derrickmehaffy** [comment]
Use extensions and add it yourself?

You could take a look at the core-api in the `strapi` package and add in the route, controller, and service for it using the extensions: https://strapi.io/documentation/3.0.0-beta.x/concepts/concepts.html#extensions

Or submit a PR to add it back in. Not sure when/why it was removed
